safety/scan/command.py

```diff
@@ -288,14 +291,11 @@ def scan(ctx: typer.Context,
     target_ecosystems = ", ".join([member.value for member in ecosystems])
     wait_msg = f"Analyzing {target_ecosystems} files and environments for security findings"
 
-    import time
-
     files: List[FileModel] = []
 
     config = ctx.obj.config
 
     count = 0
-    ignored = set()
```

Comment:
I think this ignored is used later in the function, may need to add this back in